app/controllers/commitments_controller.rb

```diff
@@ -86,6 +93,11 @@ def destroy
 
   private
 
+  def criterium_id_valid?
+    criterium = Criterium.find_by(id: params[:criterium_id])
+    criterium && criterium.commitment.nil? && criterium.user_id && criterium.user_id == current_user.id
```

Comment:
Is it worth using a `try` or two here to make it a little more concise? E.g. `criterium.try(:user_id) == current_user.id` Not sure if there's a way to get the first half `try` or `&.`-y given the use of `nil?`